tensorflow-core/tensorflow-core-api/src/test/java/org/tensorflow/op/core/ShapeOpsTest.java

```diff
@@ -0,0 +1,305 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.tensorflow.keras.utils;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/***
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+****/
+
+
+import org.tensorflow.Graph;
+import org.tensorflow.Operand;
+import org.tensorflow.Session;
+import org.tensorflow.Tensor;
+import org.tensorflow.op.Scope;
+import org.tensorflow.op.core.Constant;
+import org.tensorflow.op.core.Reshape;
+import org.tensorflow.op.core.Shape;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.TInt32;
+import org.tensorflow.types.TInt64;
+
+/**
+ *
+ * @author jbclarke
+ */
+@RunWith(JUnit4.class)
+public class ShapeOpsTest {
+    
+    public ShapeOpsTest() {
+    }
+    
+
+    /**
+     * Test of create method, of class ShapeOps.
+     */
+    @Test
+    public void testCreate_Scope() {
+        try (Graph g = new Graph();
+                Session sess = new Session(g)) {
+            Scope scope = new Scope(g);
+            ShapeOps stf = ShapeOps.create(scope);
+            assertEquals(scope, stf.scope());
+            assertEquals(TInt32.DTYPE, stf.datatype());
+        }
+    }
+
+    /**
+     * Test of create method, of class ShapeOps.
+     */
+    @Test
+    public void testCreate_Scope_DataType() {
+        System.out.println("create DataType");
```

Comment:
Given the number of tests that are in TF at the moment, having it print out the name of each one is a little much.

Reply:
Ok I will remove the println's